src/ldc/eh/common.d

```diff
@@ -14,6 +14,14 @@ import core.stdc.stdarg;
 
 import ldc.eh.fixedpool;
 
+version (ARM)
+{
+    version (iOS)
+        version = SjLj_Exceptions;
+    else
+        version = ARM_EABI_UNWINDER;
```

Comment:
ARM_EABI_UNWINDER perhaps is not used in this module.  There is a `version(ARM)` block later in this module that I did not change because I was unsure.  Should it be switched to ARM_EABI_UNWINDER?